fe/fe-core/src/main/java/com/starrocks/alter/Alter.java

```diff
@@ -188,6 +199,157 @@ public void processDropMaterializedView(DropMaterializedViewStmt stmt) throws Dd
             db.writeUnlock();
         }
     }
+    public void processAlterMaterializedView(AlterMaterializedViewStatement stmt)
+            throws DdlException, MetaNotFoundException, AnalysisException {
+        // check db
+        final TableName mvName = stmt.getMvName();
+        final String oldMvName = mvName.getTbl();
+        final String newMvName = stmt.getNewMvName();
+        final RefreshSchemeDesc refreshSchemeDesc = stmt.getRefreshSchemeDesc();
+        String dbName = mvName.getDb();
+        Database db = GlobalStateMgr.getCurrentState().getDb(dbName);
+        if (db == null) {
+            ErrorReport.reportDdlException(ErrorCode.ERR_BAD_DB_ERROR, dbName);
+        }
+
+        db.writeLock();
+        try {
+            MaterializedView materializedView = null;
+            final Table table = db.getTable(oldMvName);
+            if (table instanceof MaterializedView) {
+                materializedView = (MaterializedView) table;
+            }
+            if (materializedView == null) {
+                for (Table t : db.getTables()) {
+                    if (t instanceof MaterializedView) {
+                        if (t.getName().equals(oldMvName)) {
+                            materializedView = (MaterializedView) t;
+                            break;
+                        }
+                    }
+                }
+            }
+
+            if (materializedView == null) {
+                throw new MetaNotFoundException("Materialized view " + mvName + " is not find");
+            }
+            // check materialized view state
+            if (materializedView.getState() != OlapTableState.NORMAL) {
+                throw new DdlException("Materialized view [" + materializedView.getName() + "]'s state is not NORMAL. "
+                        + "Do not allow doing ALTER ops");
+            }
+
+            EditLog editLog = GlobalStateMgr.getCurrentState().getEditLog();
+            //rename materialized view
+            if (newMvName != null) {
+                processRenameMaterializedView(oldMvName, newMvName, db, materializedView, editLog);
```

Comment:
editLog is from GlobalStateMgr and is not need to pass argument, when you use where you can get.